### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by pepdbm7**

### CODE REVIEW

**Comment by pepdbm7** [comment]
added app-template folder and 8 html files

**Comment by pepdbm7** [comment]
added app-template folder with 8 html files

File: staff/pep-delbano/app-template/contact.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Contact</title>
+</head>
+<body>
+    <header>
+        <h1>Contact with us</h1>
+    </header>
+    <article>
```
Comment (manuelbarzi) [inline staff/pep-delbano/app-template/contact.html]: why `article` here? no sense... please, RTFM

File: staff/pep-delbano/app-template/profile.html [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    <nav><!--contains a list as it'll be a side nav-->
+        <ul>
+            <li><a href="#">EXPERIENCE</a></li>
+            <li><a href="#">SKILLS</a></li>
+            <li><a href="#">REFENCES</a></li>
+            <li><a href="#">PROJECTS</a></li>
+        </ul>
+    </nav>
+
+    <article>
```
Comment (manuelbarzi) [inline staff/pep-delbano/app-template/profile.html]: `article` here again does not seem to have sense, RTFM

File: staff/pep-delbano/app-template/profile.html [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+            <h3>CSS3</h3>
+            <progress value="70" max="100"></progress>
+        </section>
+        <section>
+            <h3>JS</h3>
+            <progress value="60" max="100"></progress>
+        </section>
+    </article>
+
+    <article>
```
Comment (manuelbarzi) [inline staff/pep-delbano/app-template/profile.html]: same here and in the other next blocks... may you just split blocks with sections instead?

**Pull request merged**